app/assets/stylesheets/application.css

```diff
@@ -18,3 +18,9 @@
     width: 400px;
     height: 400px;
 }
+
+#order-icon {
```

Comment:
if you want to can implement a new solution to stick the order icon to the right, even because this way it gives problem even if we use the message later on.
I added pull-right in the class of the navbar-nav(see below) and then I declared like this in the stylesheet:
```.pull-right {
  float: right !important;
}```